examples/init_queue.c

```diff
@@ -24,18 +24,20 @@
 #include "portals4_bxiext.h"
 
 /*
- * This example initalize an interface handle to use, allocate a buffer for incoming events
- * and perform an LEAppend to generate an event in the allocated buffer.
+ * This example initalize an interface handle to use, allocate an event queue,
+ * and perform an LEAppend to allocate memory for the specified le.

```

Comment:
missing 's'
I would suggest: "This example initializes a logical interface, allocates an event queue, and performs a LEAppend to expose to the network a portion of the process address space."